### ISSUE TITLE
oh-my-posh@18.12.0: hash check failed

### ISSUE THREAD

**Issue by acethical** [ScoopInstaller/Main#5171]
Installer blocked by Windows Security

`scoop status`

> Name       Installed Version Latest Version Missing Dependencies Info
> ----       ----------------- -------------- -------------------- ----
> oh-my-posh 18.11.0           18.12.0

`scoop cache rm oh-my-posh`
`scoop update oh-my-posh`

> oh-my-posh: 18.11.0 -> 18.12.0
> Updating 1 outdated apps:
> Updating 'oh-my-posh' (18.11.0 -> 18.12.0)
> Downloading new version
> themes.zip (127.5 KB) [============================================================================================================================================================================================================] 100%
> Checking hash of themes.zip ... ok.
> posh-windows-amd64.exe (16.0 MB) [=================================================================================================================================================================================================] 100%
> Checking hash of posh-windows-amd64.exe ... Get-FileHash: C:\Users\cat\scoop\apps\scoop\current\lib\install.ps1:674
> Line |
>  674 |      $actual = (Get-FileHash -Path $file -Algorithm $algorithm).Hash.T …
>      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>      | Operation did not complete successfully because the file contains a virus or potentially unwanted software. :
>      | 'C:\Users\cat\scoop\cache\oh-my-posh#18.12.0#https_github.com_JanDeDobbeleer_oh-my-posh_releases_download_v18.12.0_posh-windows-amd64.exe_oh-my-posh.exe'
> InvalidOperation: C:\Users\cat\scoop\apps\scoop\current\lib\install.ps1:674
> Line |
>  674 |      $actual = (Get-FileHash -Path $file -Algorithm $algorithm).Hash.T …
>      |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>      | You cannot call a method on a null-valued expression.
> Get-Content: C:\Users\cat\scoop\apps\scoop\current\lib\core.ps1:1173
> Line |
> 1173 |          return Get-Content $file -AsByteStream -TotalCount 8
>      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>      | Operation did not complete successfully because the file contains a virus or potentially unwanted software. :
>      | 'C:\Users\cat\scoop\cache\oh-my-posh#18.12.0#https_github.com_JanDeDobbeleer_oh-my-posh_releases_download_v18.12.0_posh-windows-amd64.exe_oh-my-posh.exe'
> ERROR Hash check failed!
> App:         main/oh-my-posh
> URL:         https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v18.12.0/posh-windows-amd64.exe#/oh-my-posh.exe
> First bytes:
> Expected:    262e22954806807a9404131a3837c71c1732781f4c8c8b09120fbbb838787928
> Actual:
> 
> Please try again or create a new issue by using the following link and paste your console output:
> https://github.com/ScoopInstaller/Main/issues/new?title=oh-my-posh%4018.12.0%3a+hash+check+failed

<img width="344" alt="ApplicationFrameHost_SUYCC9SOzb" src="https://github.com/ScoopInstaller/Main/assets/10352574/c2088cc1-fd1c-4f6c-ae11-565b22caf556">
